src/appleseed/renderer/modeling/color/colorspace.h

```diff
@@ -34,10 +34,18 @@
 namespace renderer
 {
 
+
+//
+// Standard lighting conditions (D65 illuminant, CIE 1931 2-deg color matching functions).
```

Comment:
The previous comment stated the use of CIE 1964 10-deg CMFs, now your comments states the use of CIE 1931 2-deg CMFs. Did you change the standard lighting conditions?

Reply:
The comment was wrong

Reply:
Nice catch!